src/Nirum/Targets/Python.hs

```diff
@@ -1007,17 +1018,10 @@ class $className(object):
         ]
 compileTypeDeclaration src
                        d@TypeDeclaration { typename = typename'
-                                         , type' = UnionType tags
+                                         , type' = u
```

Comment:
We don't need to define `tags` like the following code in `where` clause:

```haskell
tags :: [Tag]
tags = findTags u
```

Instead we can just match both of `UnionType` and its `tags`:

```haskell
, type' = u@(UnionType tags)
```

Also we should rename `u` to better one.